src/pip/_internal/download.py

```diff
@@ -25,12 +25,13 @@
 from pip._vendor.six.moves import xmlrpc_client  # type: ignore
 from pip._vendor.six.moves.urllib import parse as urllib_parse
 from pip._vendor.six.moves.urllib import request as urllib_request
-from pip._vendor.urllib3.util import IS_PYOPENSSL
 
 import pip
 from pip._internal.exceptions import HashMismatch, InstallationError
 from pip._internal.locations import write_delete_marker_file
 from pip._internal.models.index import PyPI
+# Import ssl from search_scope so the initial import occurs in only one place.
+from pip._internal.models.search_scope import HAS_TLS, ssl
```

Comment:
I'd move it to `src/pip/_internal/utils/compat.py`, just like `WINDOWS`

Reply:
Good suggestion, thanks! And done.